projects/core/src/auth/services/open-id-token/open-id-token.service.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { AuthConfig } from '../../config/auth-config';
+import { OpenIdToken } from '../../models/token-types.model';
+
+const OAUTH_ENDPOINT = '/authorizationserver/oauth/token';
+
+@Injectable()
+export class OpenIdAuthenticationTokenService {
+  constructor(private config: AuthConfig, private http: HttpClient) {}
+
+  loadOpenIdAuthenticationToken(
+    username: string,
+    password: string
+  ): Observable<OpenIdToken> {
+    const url = this.getOAuthEndpoint();
+    const params = new HttpParams()
+      .set('client_id', this.config.authentication.kyma_client_id)
```

Comment:
Shall we encodeuri these in a similar way than client-authentication-token.service ?

Reply:
Done